crates/args/src/lib.rs

```diff
@@ -13,6 +13,29 @@ pub struct Args {
     #[arg(long, default_value_t = 9002, env = "ETHUI_WS_PORT")]
     pub ws_port: u16,
 
-    #[arg(long, default_value_t = false)]
-    pub hidden: bool,
+    #[command(subcommand)]
+    pub command: Option<Commands>,
+}
+
+#[derive(Subcommand, Debug)]
+pub enum Commands {
+    #[command(name = "app")]
+    App {
+        #[arg(long, default_value_t = false)]
+        hidden: bool,
+    },
+
+    #[cfg(feature = "forge-traces")]
+    /// Run forge tests
+    #[command(name = "forge")]
+    Forge {
+        #[command(subcommand)]
+        subcommand: ForgeCommands,
+    },
+}
+
+#[derive(Subcommand, Debug)]
+pub enum ForgeCommands {
+    /// Run tests
+    Test,
```

Comment:
Ideally, we should be able to receive arbitrary arguments here, and forward them all to forge test

e.g.: `ethui forge test --match-contract ...`
we'd only need to enforce adding `-vvvvv --json` which are the ones we do need